`test_out` may fail due to slightly different values caused by different order of matrizes in SGEMM:

> Mismatched elements: 1 / 50 (2.0%)
> Greatest absolute difference: 1.430511474609375e-05 at index (4, 5) (up to 1e-05 allowed)
> Greatest relative difference: 4.65393206065873e-06 at index (4, 5) (up to 1.3e-06 allowed)

Author: Alexander Grund (TU Dresden)
Updated for PyTorch 1.13.1: Simon Branford (University of Birmingham)

--- a/test/test_ops.py
+++ b/test/test_ops.py
@@ -545,6 +545,9 @@
             else list(supported_dtypes)[0]
         )
 
+        if dtype is torch.float32:
+            self.precision, self.rel_tol = (1.5e-05, 1e-05)
+
         samples = op.sample_inputs(device, dtype)
         for sample in samples:
             # calls it normally to get the expected result
